package.json

```diff
@@ -67,6 +67,7 @@
   "scripts": {
     "test": "grunt test",
     "e2e": "./node_modules/.bin/codeceptjs run --steps",
+    "e2e-debug": "./node_modules/.bin/codeceptjs run --steps --debug --verbose",
```

Comment:
I considered that. If only developers run these tests it might be worth changing, I was considering when simply running tests (rather than writing them) that the verbose output could be distracting/problematic...

Reply:
This is a dev only thing right now, I think. When others use this they'll most likely look at the generated report. If that ends up happening we might be better off making the default behavior more verbose and and creating a new script later that runs headless and pops up a report afterward.